### PR TITLE
Addition: Pull down to refresh in nearby activity

### PR DESCRIPTION
**Pull request by 97balakrishnan**
PR for #264 
Pull down to refresh feature added in nearby activity.

**SwipeRefreshLayout** used

### CODE REVIEW

File: app/src/main/res/layout/activity_nearby.xml [hunk 1]
```diff
@@ -22,16 +22,26 @@
...
             android:layout_below="@id/toolbar">
 
+
             <ProgressBar
                 android:id="@+id/progressBar"
                 android:layout_width="match_parent"
                 android:layout_height="wrap_content" />
 
+            <android.support.v4.widget.SwipeRefreshLayout
+
```
Comment (maskaravivek) [inline app/src/main/res/layout/activity_nearby.xml]: remove blank lines

File: app/src/main/java/fr/free/nrw/commons/nearby/NearbyActivity.java [hunk 2]
```diff
@@ -70,6 +71,14 @@ protected void onCreate(Bundle savedInstanceState) {
         bundle = new Bundle();
         initDrawer();
         initViewState();
+        swipeLayout=(SwipeRefreshLayout)findViewById(R.id.swipe_container);
```
Comment (maskaravivek) [inline app/src/main/java/fr/free/nrw/commons/nearby/NearbyActivity.java]: use butterknife. 

Refer to: https://github.com/JakeWharton/butterknife

**Comment by 97balakrishnan** [comment]
Sure. I will make the changes and commit ASAP.

**Comment by 97balakrishnan** [comment]
Please review the changes

**Comment by maskaravivek** [comment]
@97balakrishnan Swipe to refresh works fine. The list updates even if the location has not changed. 

@misaochan @neslihanturan Is that behaviour fine. If yes, we can go ahead and merge this. Have tested it on a real device.

**Comment by maskaravivek** [comment]
I just noticed that it exactly replicates the functionality of the refresh button. 

Going ahead and merging it. @97balakrishnan congrats for your first contribution. :)

**Comment by 97balakrishnan** [comment]
Thanks a lot. My pleasure. I'll make the changes to prevent it from refreshing if the locations not changed ASAP. Thanks again

**Pull request merged**